openwisp_controller/config/models.py

```diff
@@ -119,7 +119,8 @@ class Meta(AbstractConfig.Meta):
 
     def clean(self):
         if not hasattr(self, 'organization') and self._has_device():
-            self.organization = self.device.organization
+            if hasattr(self.device, 'organization'):
```

Comment:
This nested if can be avoided in this case, please rewrite it to:

```python
if all([
    not hasattr(self, 'organization'),
    self._has_device(),
    hasattr(self.device, 'organization')
]):
    self.organization = self.device.organization
```

Reply:
Ok. will do that immediately